StripeConnect/StripeConnectTests/Internal/Webview/MessageHandlers/OnLoadErrorMessageHandlerTests.swift

```diff
@@ -4,26 +4,46 @@
 //
 //  Created by Chris Mays on 8/14/24.
 //
-@testable import StripeConnect
+@_spi(PrivateBetaConnect) @testable import StripeConnect
 import XCTest
 
 class OnLoadErrorMessageHandlerTests: ScriptWebTestBase {
 
     @MainActor
     func testMessageSend() async throws {
-        let expectation = self.expectation(description: "Message received")
```

Comment:
expectation isn't needed because `evaluateOnLoadError` is async and we already await its completion